pyproject.toml

```diff
@@ -1,39 +1,218 @@
-[tool.poetry]
+[project]
 name = "dataclasses-json"
-version = "0.0.0"
+dynamic = ["version"]
 description = "Easily serialize dataclasses to and from JSON."
-authors = ["Charles Li <charles.dt.li@gmail.com>"]
-maintainers = ['Charles Li <charles.dt.li@gmail.com>', 'Georgiy Zubrienko <gzu@ecco.com>', 'Vitaliy Savitskiy <visa@ecco.com>', 'Matthias Als <mata@ecco.com>']
-license = 'MIT'
+authors = [{ name = "Charles Li", email = "charles.dt.li@gmail.com" }]
+requires-python = ">=3.9"
```

Comment:
You can drop 3.9 https://devguide.python.org/versions/#status-of-python-versions